dell-csi-helm-installer/verify.sh

```diff
@@ -449,6 +449,61 @@ function verify_helm_values_version() {
   check_error error
 }
 
+# verify that the authorization sidecar is configured correctly to be used with the authorization proxy server
+function verify_authorization_proxy_server() {
+  if [ ${NODE_VERIFY} -eq 0 ]; then
+    return
+  fi
+
+  log step "Verifying csm-authorization connectivity"
+
+  enabled=$(grep -v "#" $VALUES | grep -A1 "authorization:" | grep enabled | xargs | awk '{print $2}')
+  if [ "${enabled}" == "false"  ]; then
+    return
+  fi
+
+  proxyHost=$(grep -v "#" $VALUES | grep proxyHost | xargs | awk '{print $2}')
+  insecure=$(grep -v "#" $VALUES | grep -A10 "authorization:" | grep insecure | xargs | awk '{print $2}')
+  WGET=`which wget`
```

Comment:
This command and others (line 468, 484, 486, 493, and 495) should be run on the worker nodes and not the node where the installation is being run.

Reply:
From a[ comment](https://github.com/dell/csi-powerscale/pull/21/files#r741765203) from the similar PowerScale PR:

"This is a common file across all the drivers and comes from https://eos2git.cec.lab.emc.com/DevCon/dell-csi-helm-installer/tree/master/installer . So please make these changes in git@eos2git.cec.lab.emc.com:DevCon/dell-csi-helm-installer.git, and then sync the same with driver repos. Please refer the README for instructions.

Once the PR at git@eos2git.cec.lab.emc.com:DevCon/dell-csi-helm-installer.git is merged and synced with driver repo, then this PR can be merged."

So I am going to remove this file for now. I'll make sure these commands are run against the nodes instead when it gets merged back into here.